package.json

```diff
@@ -21,6 +21,8 @@
     "format": "prettier --write .",
     "docker:start": "./scripts/launch_server.sh",
     "docker:stop": "docker compose down",
+    "docker:remote-be:start": "docker compose -f remote-backend/docker-compose.yml up --detach",
```

Comment:
These are used if you are pointing your local to TW Staging or Prod.

Reply:
Nice, I always use it but changing to remote-backend folder!